### PR TITLE
modify device-plugins.md

### PR DESCRIPTION
**Pull request by Yvonnehrr**
Spelling mistake.  "co-ordintated" should be "co-ordinated".

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@Yvonnehrr
Pls sign the CLA in order for the PR to be reviewed.

**Comment by bradtopol** [comment]
@Yvonnehrr Thank you for your contribution.  Please sign the CLA so we can accept your PR for review.

**Comment by Yvonnehrr** [comment]
I signed it.

**Comment by Yvonnehrr** [comment]
I signed it
/retest
/check-cla

**Comment by zparnold** [comment]
Hi there I tried rechecking the CLA, is the author email associated with your commit the same one you signed the CLA with?

**Comment by Yvonnehrr** [comment]
> Hi there I tried rechecking the CLA, is the author email associated with your commit the same one you signed the CLA with?

It's strange that i signed in with "Yvonnehrr"，but it reminds me "h4ghhh needs CLA". Though “h4ghhh” is also mine.

**Comment by kbhawkey** [comment]
> 
> 
> > Hi there I tried rechecking the CLA, is the author email associated with your commit the same one you signed the CLA with?
> 
> It's strange that i signed in with "Yvonnehrr"，but it reminds me "h4ghhh needs CLA". Though “h4ghhh” is also mine.

Hello @Yvonnehrr . It is my understanding that the email accounts must be the same. Would you double check your CLA?
Thanks!

File: content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -210,7 +210,7 @@ Support for the "PodResources service" requires `KubeletPodResources` [feature g
 
 {{< feature-state for_k8s_version="v1.18" state="beta" >}}
 
-The Topology Manager is a Kubelet component that allows resources to be co-ordintated in a Topology aligned manner. In order to do this, the Device Plugin API was extended to include a `TopologyInfo` struct.
+The Topology Manager is a Kubelet component that allows resources to be co-ordinated in a Topology aligned manner. In order to do this, the Device Plugin API was extended to include a `TopologyInfo` struct.
```
Comment (kbhawkey) [inline content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:213]: Hello @Yvonnehrr .
Thanks for this pull request. One suggested edit:
```suggestion
The Topology Manager is a kubelet component that allows resources to be coordinated in a Topology aligned manner. In order to do this, the Device Plugin API was extended to include a `TopologyInfo` struct.
```

Comment (Yvonnehrr) [reply content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:213]: Thanks for your suggest.  I switch to another account to solve email confict.

**Pull request closed without merging**